### PR TITLE
[RO] Add `Help centre/Website`

### PR DESCRIPTION
**Pull request by 28edx**
Translation for the `Website` article.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Website/ro.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## Probleme comune {id=common-issues}
+
+### Nu am primit niciun răspuns la tichetul meu de asistență și au trecut 2 săptămâni! {id=support-no-reply}
+
+**În marea majoritatea timpului, acest lucru se datorează numărului mare de tichete de asistență care intră în sistemul nostru în același timp.**
+
+Poți să trimiți oricând un singur răspuns tichetului tău deschis de asistență pentru a cere starea cazului tău, dar asigură-te că nu faci asta mai mult de o dată sau de două ori pe săptămână. Poți să ne contactezi și pe Twitter la [@osusupport](https://twitter.com/osusupport) pentru a întreba despre tichetul tău.
+
+În cazuri rare, tichetele de asistență de la utilizatorii care comit abateri în mod repetat pot fi ignorate, deși [echipa de asistență](/wiki/People/The_Team/Account_support_team) te va anunța **întotdeauna** când nu îți va mai răspunde prima dată. Dacă nu ai primit o notificare de genul, nu trebuie să te îngrijorezi!
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:34]:
```suggestion
În cazuri rare, tichetele de asistență de la utilizatorii care realizează abateri în mod repetat pot fi ignorate, deși [echipa de asistență](/wiki/People/The_Team/Account_support_team) te va anunța **garantat** prima dată când nu îți va mai răspunde. Dacă nu ai primit o notificare de genul, nu trebuie să te îngrijorezi!
```

File: wiki/Help_centre/Website/ro.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Pot să blochez complet un alt utilizator să mă contacteze? {id=ignore-user}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:36]:
```suggestion
### Pot să blochez complet un alt utilizator din a mă contacta? {id=ignore-user}
```

File: wiki/Help_centre/Website/ro.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+**Da, absolut.**
+
+În cazul în care un alt utilizator te agresează prin mesaje private, fie pe forum sau prin clientul jocului, sunt câțiva pași pe care îi poți urma pentru a filtra mesajele lor.
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:40]:
```suggestion
În cazul în care un alt utilizator te agresează prin mesaje private, fie pe forum sau prin client-ul jocului, există câțiva pași pe care îi poți urma pentru a filtra mesajele lor.
```

File: wiki/Help_centre/Website/ro.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+**Pentru a bloca un utilizator de a îți mai trimite mesaje în joc, pur și simplu adaugă-i pe lista persoanelor blocate.**
+
+1. Fă clic pe butonul de `Options` din joc.
+2. Scrie `ignore` pentru a filtra opțiunea în secțiunea `Lista persoanelor blocate pe chat(separate prin spațiu)`.
+3. Adaugă numele de utilizator respectiv pe listă, separând fiecare utilizator nou cu un spațiu și înlocuind orice spațiu în numele lor cu underscore (Un utilizator numit *The Gatekeeper* devine *The\_Gatekeeper*, de exemplu).
+
+Nu vei mai vedea mesaje publice de la niciun utilizator din lista respectivă sau mesaje private de la ei.
+
+Dacă dorești să ignori mesaje evidențiate, mesaje private sau mesaje publice, poți adăuga pur și simplu `@h`, `@p`, `@c` la numele lor de utilizator. `The_Gatekeeper@c` va ignora mesajele utilizatorului în canalele publice. Poți să le combini: `The_Gatekeeper@ph` va ignora mesajele evidențiate și mesajele private, dar mesajele publice rămân vizibile.
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:52]:
```suggestion
Dacă dorești să ignori mesaje evidențiate, mesaje private sau mesaje publice, poți adăuga `@h`, `@p`, `@c` la numele lor de utilizator. `The_Gatekeeper@c` va ignora mesajele utilizatorului în canalele publice. De asemenea, poți să le combini astfel: `The_Gatekeeper@ph` va ignora mesajele evidențiate și mesajele private, dar mesajele publice rămân vizibile.
```

File: wiki/Help_centre/Website/ro.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Blocarea tuturor străinilor de a trimite mesaje private în joc {id=ignore-strangers-in-game}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:54]:
```suggestion
#### Blocarea tuturor străinilor din a trimite mesaje private în joc {id=ignore-strangers-in-game}
```

File: wiki/Help_centre/Website/ro.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+În cazuri rare, tichetele de asistență de la utilizatorii care comit abateri în mod repetat pot fi ignorate, deși [echipa de asistență](/wiki/People/The_Team/Account_support_team) te va anunța **întotdeauna** când nu îți va mai răspunde prima dată. Dacă nu ai primit o notificare de genul, nu trebuie să te îngrijorezi!
+
+### Pot să blochez complet un alt utilizator să mă contacteze? {id=ignore-user}
+
+**Da, absolut.**
+
+În cazul în care un alt utilizator te agresează prin mesaje private, fie pe forum sau prin clientul jocului, sunt câțiva pași pe care îi poți urma pentru a filtra mesajele lor.
+
+#### Blocarea unui utilizator de a te mai contacta în joc. {id=ignore-user-in-game}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:42]:
```suggestion
#### Blocarea unui utilizator din a te mai contacta în joc. {id=ignore-user-in-game}
```

File: wiki/Help_centre/Website/ro.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+3. Adaugă numele de utilizator respectiv pe listă, separând fiecare utilizator nou cu un spațiu și înlocuind orice spațiu în numele lor cu underscore (Un utilizator numit *The Gatekeeper* devine *The\_Gatekeeper*, de exemplu).
+
+Nu vei mai vedea mesaje publice de la niciun utilizator din lista respectivă sau mesaje private de la ei.
+
+Dacă dorești să ignori mesaje evidențiate, mesaje private sau mesaje publice, poți adăuga pur și simplu `@h`, `@p`, `@c` la numele lor de utilizator. `The_Gatekeeper@c` va ignora mesajele utilizatorului în canalele publice. Poți să le combini: `The_Gatekeeper@ph` va ignora mesajele evidențiate și mesajele private, dar mesajele publice rămân vizibile.
+
+#### Blocarea tuturor străinilor de a trimite mesaje private în joc {id=ignore-strangers-in-game}
+
+Dacă dorești să împiedici persoanele care nu se află pe lista ta de prieteni să îți trimită mesaje private, urmează următorii pași. **Ține cont: moderatorii nu sunt afectați de această funcționalitate și pot oricând să îți trimită mesaje private. Dacă crezi că un moderator te agresează în ciuda acestui lucru, trebuie să contactezi [echipa noastră de asistență](mailto:support@ppy.sh).**
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:56]:
```suggestion
Dacă dorești să împiedici persoanele care nu se află pe lista ta de prieteni din a îți trimite mesaje private, urmează următorii pași. **Ține cont: moderatorii nu sunt afectați de această funcționalitate și pot oricând să îți trimită mesaje private. Dacă crezi că un moderator te agresează în ciuda acestui lucru, trebuie să contactezi [echipa noastră de asistență](mailto:support@ppy.sh).**
```

File: wiki/Help_centre/Website/ro.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Fă clic pe butonul de `Options` în meniul principal sau apasă `Ctrl` + `O`.
+2. Scrie `block` pentru a filtra opțiunea în secțiunea de `Mesaje în cadrul jocului`.
+3. Fă clic pe setarea care apare pentru a o activa.
+
+Utilizatorii care nu sunt pe lista ta de prieteni nu vor mai putea să te contacteze.
+
+#### Împiedică mesajele private noi de pe forum să ajungă la tine. {id=ignore-strangers-website}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:64]:
```suggestion
#### Împiedică mesajele private noi de pe forum din a ajunge la tine. {id=ignore-strangers-website}
```

File: wiki/Help_centre/Website/ro.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+Du-te în [setările contului](https://osu.ppy.sh/home/account/edit) și fă clic pe `blochează mesajele private de la oameni care nu sunt pe lista ta de prieteni` în secțiunea de Confidențialitate. Mesajele private de la utilizatorii de pe lista ta de prieteni vor putea să îți trimită mesaje.
+
+### Și dacă încă întâmpin probleme cu un utilizator după ce l-am blocat? {id=ignore-did-not-help}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:68]:
```suggestion
### Ce fac dacă încă întâmpin probleme cu un utilizator după ce l-am blocat? {id=ignore-did-not-help}
```

File: wiki/Help_centre/Website/ro.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+**Fără griji, echipa noastră de asistență te va ajuta.**
+
+Dacă un utilizator încă te agresează, fie prin alte mijloace sau își folosește prietenii sau pe altcineva pentru a trece de eforturile tale de a îl bloca, te rugăm să ne trimiți un e-mail la [support@ppy.sh](mailto:support@ppy.sh).
+
+Asigură-te că incluzi orice jurnal cu mesaje sau alte dovezi în e-mail, iar echipa noastră de asistență va investiga cazul tău și îl va remedia pentru tine.
+
+### Ce sunt aceste pagini „Eu!” pe care le văd pe profilul altor utilizatori? {id=userpage}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:76]:
```suggestion
### Ce sunt aceste pagini „eu!” pe care le văd pe profilul altor utilizatori? {id=userpage}
```

File: wiki/Help_centre/Website/ro.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+**Paginile Eu! sau pagini de utilizator sunt elemente speciale de profil la care au acces orice utilizator cu [suporter](https://osu.ppy.sh/home/support). Ele pot conține orice vrei — poze cu realizările tale, beatmap-urile tale, practic tot ce îți dorești!**
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:78]:
```suggestion
**Paginile „eu!” (sau paginile de utilizator) sunt elemente speciale de profil la care au acces orice [suporter](https://osu.ppy.sh/home/support). Ele pot conține orice dorești — poze cu realizările tale, beatmap-urile tale, practic tot ți-ai putea imagina!**
```

File: wiki/Help_centre/Website/ro.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+
+Jucătorii cu [suporter](https://osu.ppy.sh/home/support) au privilegiul de a-și modifica propria lor pagină pentru a fi afișată pe profilul lor, pentru a fi văzut de toată lumea. Paginile Eu! pot conține orice vrei, atâta timp cât respectă [regulile comunității](/wiki/Rules)!
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:80]:
```suggestion
Jucătorii cu statusul de [suporter](https://osu.ppy.sh/home/support) au privilegiul de a-și modifica propria lor pagină pentru a fi afișată pe profilul lor pentru a fi văzută de toată lumea. Paginile „eu!” pot conține orice vrei, atâta timp cât respectă [regulile comunității](/wiki/Rules)!
```

File: wiki/Help_centre/Website/ro.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+Chiar dacă se întâmplă ca [suporterul](https://osu.ppy.sh/home/support) tău să se termine, pagina ta Eu! va **rămâne vizibilă** și vei putea în continuare să îi modifici conținutul.
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:82]:
```suggestion
Chiar dacă se întâmplă ca statusul tău de [suporter](https://osu.ppy.sh/home/support) să se expire, pagina ta „eu!” va **rămâne vizibilă** și vei putea în continuare să îi modifici conținutul.
```

File: wiki/Help_centre/Website/ro.md [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+
+### Conținutul paginii mele Eu! a dispărut! {id=userpage-removed}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:84]:
```suggestion
### Conținutul paginii mele „eu!” a dispărut! {id=userpage-removed}
```

File: wiki/Help_centre/Website/ro.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+
+**Acest lucru se poate întâmpla dacă pagina ta Eu! a avut conținut necorespunzător, așa cum este definit în [regulile comunității](/wiki/Rules).**
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:86]:
```suggestion
**Acest lucru se poate întâmpla dacă pagina ta „eu!” a avut conținut necorespunzător, așa cum este definit în [regulile comunității](/wiki/Rules).**
```

File: wiki/Help_centre/Website/ro.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+
+În astfel de cazuri, poți deschide un tichet cu echipa de asistență prin e-mail la [accounts@ppy.sh](mailto:accounts@ppy.sh) și să discuți situația ta cu ei.
+
+Dacă în orice clipă ai îndoieli dacă ceva este adecvat pentru o pagină Eu!, poți trimite mesaj oricăruia dintre membrii [Echipei de Moderare Globală](/wiki/People/The_Team/Global_Moderation_Team).
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:90]:
```suggestion
Dacă în orice clipă ai îndoieli legate dacă ceva este adecvat pentru o pagină „eu!”, poți trimite mesaj oricăruia dintre membrii [Echipei de Moderare Globală](/wiki/People/The_Team/Global_Moderation_Team).
```

File: wiki/Help_centre/Website/ro.md [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+
+### Trebuie să îmi folosesc adresa de e-mail pentru a mă înregistra? {id=email-purpose}
+
+**Da, avem nevoie de adresa ta de e-mail pentru a îți trimite cereri de resetare a parolei și coduri de verificare când este necesar.**
+
+Dacă întâmpini probleme cu conectarea în joc, te rugăm să verifici setările firewall și antivirus pentru a te asigura că osu! nu este blocat.
+
+Dacă îți pierzi vreodată parola sau nu îți poți accesa contul, e-mailul tău de înregistrare este primul nostru mijloc de contact cu tine.
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:98]:
```suggestion
Dacă îți pierzi vreodată parola sau nu îți poți accesa contul, e-mailul tău folosit la înregistrare este primul nostru mijloc de contact cu tine.
```

File: wiki/Help_centre/Website/ro.md [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+  - message
+  - page
+  - profile
+  - missing
+  - stats
+  - aparență
+  - mesaj
+  - pagină
+  - profil
+  - dispărut
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:17]:
```suggestion
  - dispărut
  - lipsă
```

**Review by NiceAesth — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/Website/ro.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+---
+
+# Site web
+
+*Pagina principală: [Centru ajutor](/wiki/Help_centre)*
+
+Această secțiune este dedicată problemelor cu site-ul osu!
+
+## Probleme comune {id=common-issues}
```
[the remark this answers is not in the document]
Comment (28edx) [reply wiki/Help_centre/Website/ro.md:26]: „Comune” can also mean „Firesc, frecvent, normal, obișnuit.”. wouldn't it better to stick with „frecvente” then, given the context?

File: wiki/Help_centre/Website/ro.md (hunk 13 above)
Comment (28edx) [reply wiki/Help_centre/Website/ro.md:82]: isn't it supposed to be „să expire” and not „să se expire”?

File: wiki/Help_centre/Website/ro.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (NiceAesth) [reply wiki/Help_centre/Website/ro.md:27]: would still prefer this over the other two in formal settings

File: wiki/Help_centre/Website/ro.md [hunk 20]
```diff
@@ -0,0 +1,105 @@
...
+
+# Site web
+
+*Pagina principală: [Centru ajutor](/wiki/Help_centre)*
+
+Această secțiune este dedicată problemelor cu site-ul osu!
+
+## Probleme uzuale {id=common-issues}
+
+### Nu am primit niciun răspuns la tichetul meu de asistență și au trecut 2 săptămâni! {id=support-no-reply}
```
Comment (NiceAesth) [inline wiki/Help_centre/Website/ro.md:29]:
```suggestion
### Nu am primit nici un răspuns la tichetul meu de asistență și au trecut 2 săptămâni! {id=support-no-reply}
```

**Review by NiceAesth — CHANGES_REQUESTED** [review]
missed this one, once fixed the rest is ok

**Review by NiceAesth — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**